tests/contrib.bats

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "bud with a path to a Dockerfile (-f) containing a non-directory entry" {
```

Comment:
This description doesn't look right for what this test is doing.

Reply:
Lets let it run through, I think @edsantiago is going to have some ideas on how better to handle this.  Perhaps copy the Dockefile into tests and then modify the FROM line.